pyfixest/estimation/feiv_.py

```diff
@@ -133,6 +139,28 @@ def get_fit(self) -> None:
         _Z = self._Z
         _Y = self._Y
 
+        #  Start First Stage
+
+        model1 = Feols(
+            Y=_X[:, -1].reshape(-1, 1),
+            X=_Z,
+            weights=np.ones(self._weights.shape[0]).reshape(-1, 1),
```

Comment:
Here we do not want to use `self._weights` because the weights are already applied to `_X` and `_Z`, right? Nice catch =)

Reply:
Yes, this is exactly what I had in mind!